lib/commons/aria/index.js

```diff
@@ -194,24 +212,24 @@ lookupTables.role = {
 		context: null,
 		implicit: ['button', 'input[type="button"]', 'input[type="image"]']
 	},
-	'checkbox': {
-		type: 'widget',
-		attributes:  {
-			required: ['aria-checked']
+	'cell': {
+		type: 'structure',
+		attributes: {
+			allowed: ['aria-colindex', 'aria-colspan', 'aria-rowindex', 'aria-rowspan']
 		},
 		owned: null,
 		nameFrom: ['author', 'contents'],
-		context: null,
-		implicit: ['input[type="checkbox"]']
+		context: ['row']
 	},
-	'switch': {
```

Comment:
I just moved switch down to preserve the alphabetical order in this file. Hope that's OK.